src/main/java/com/connexta/store/adaptors/S3StorageAdaptor.java

```diff
@@ -132,4 +161,12 @@ private S3Object getS3Object(@NotBlank final String key) throws RetrieveExceptio
 
     return s3Object;
   }
+
+  /** The caller is responsible for closing the returned {@link InputStream}. */
+  @NotNull
+  public InputStream retrieveFileStream(@NotBlank final String key) throws StoreException {
```

Comment:
This method isn't in the StorageAdaptor interface, so it can never be called

Reply:
removed as per your previous comment